tokens/README.md

```diff
@@ -125,3 +125,115 @@ recipient address. That SOL can be used to pay transaction fees on staking
 operations such as delegating stake. The rest of the allocation is put in
 a stake account. The new stake account address is output in the transaction
 log.
+
+## Distribute SPL tokens
+
+Distributing SPL Tokens works very similarly to distributing SOL, but requires
+the `--owner` parameter to sign transactions. Each recipient account must be an
+system account that will own an Associated Token Account for the SPL Token mint.
+The Associated Token Account will be created, and funded by the fee_payer, if it
+does not already exist.
+
+Send SPL tokens to the recipients in `<RECIPIENTS_CSV>`.
+
+Example recipients.csv:
+
+```text
+recipient,amount
+CYRJWqiSjLitBAcRxPvWpgX3s5TvmN2SuRY3eEYypFvT,75.4
```

Comment:
wydt about not using ui_amount here?  When I wrote my CVS independently, I used the u64 amount because that's what felt more natural for program -> program communication.  I feel this is like SOL vs lamports: ui amount for users, u64 amount for programs

Reply:
I've adapted my code to use ui_amount so 🤷🏼‍♂️

Reply:
I felt like ui amount was more consistent with the current `solana-tokens`, which is generally user-facing.
But if we want to change that, now would be a good time, as I'm about to start rebasing...

Reply:
I'm ok with ui amount